packages/driver/src/cross-origin/patches/cookies.js

```diff
@@ -1,3 +1,46 @@
+const parseCookieString = (cookieString) => {
```

Comment:
I had converted it to TypeScript and then converted it back when it became evident it would need to go into the injection, thinking it couldn't be TypeScript for that. Looking into it now, it seems that's not an actual limitation of code going into the injection, so I'll convert it to TypeScript once again.